server/s3_md5_hash.cc

```diff
@@ -164,6 +164,9 @@ int MD5hash::Finalize() {
     return 0;
   }
   pi_bufvec.ov_vec.v_nr = 0;
+  // Function gets called in send_response_to_s3_client
```

Comment:
@knrajnambiar76 Lets discuss about this.

Reply:
Add Warning Log and remove this comment